grizli/prep.py

```diff
@@ -4612,7 +4809,10 @@ def visit_grism_sky(grism={}, apply=True, column_average=True, verbose=True, ext
     fig = plt.figure(figsize=[6., 6.])
     ax = fig.add_subplot(111)
 
-    im_shape = (1014, 1014)
+    if isJWST:
```

Comment:
See above, i.e., MIRI will be different, though I guess we (probably) won't be doing grism background for MIRI
```python
... 
if _instrument in ['NIRISS', 'NIRCAM']:
    ...
```